securedrop/translations/messages.pot

```diff
@@ -974,6 +986,15 @@ msgstr ""
 msgid "Sorry, we couldn't locate what you requested."
 msgstr ""
 
+#: source_templates/session_timeout.html:6
+msgid ""
+"Your session timed out due to inactivity. Please login again if you want "
+"to continue using SecureDrop, or select \"New Identity\" from the green "
+"onion button in the Tor browser to clear all history of your SecureDrop "
+"usage from this device. If you are not using Tor Browser, restart your "
```

Comment:
Do we want this message `If you're not using Tor Browser` shown to potential sources? I am not sure about the case where a user would not be using TBB on a production SD install.

Reply:
Yes, this can happen if a user is using a proxy (e.g. if they are on Qubes and proxying all traffic through a Tor network VM)